### PR TITLE
Test eslint

### PR DESCRIPTION
**Pull request by teoljungberg**

### CODE REVIEW

File: app/assets/javascripts/es6.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  }
+}
+
+class Cat extends Animal {
+  speak() {
+    console.log(this.name + " loves to chase cards and bite their tires like a dog, but isn't one.");
+  }
+}
+
+a(foo => { if (true) {}; });
```
Comment (houndci-staging) [inline app/assets/javascripts/es6.js:23]: "a" is not defined.<br>Expected parentheses around arrow function argument.<br>"foo" is defined but never used<br>Unexpected constant condition.<br>Empty block statement.<br>Unnecessary semicolon.

File: app/assets/javascripts/es6.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+class Dog extends Animal {
+  speak() {
+    console.log(this.name + ' barks.');
+  }
+}
+
+class Cat extends Animal {
+  speak() {
+    console.log(this.name + " loves to chase cards and bite their tires like a dog, but isn't one.");
```
Comment (houndci-staging) [inline app/assets/javascripts/es6.js:19]: Line 19 exceeds the maximum line length of 80.<br>Expected indentation of 6 space characters but found 4.<br>Unexpected console statement.

File: app/assets/javascripts/es6.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  }
+}
+
+class Dog extends Animal {
+  speak() {
+    console.log(this.name + ' barks.');
+  }
+}
+
+class Cat extends Animal {
```
Comment (houndci-staging) [inline app/assets/javascripts/es6.js:17]: "Cat" is defined but never used

File: app/assets/javascripts/es6.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  }
+
+  speak() {
+    console.log(this.name + ' makes a noise.');
+  }
+}
+
+class Dog extends Animal {
+  speak() {
+    console.log(this.name + ' barks.');
```
Comment (houndci-staging) [inline app/assets/javascripts/es6.js:13]: Expected indentation of 6 space characters but found 4.<br>Unexpected console statement.<br>Strings must use doublequote.

File: app/assets/javascripts/es6.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  constructor(name) {
+    this.name = name;
+  }
+
+  speak() {
+    console.log(this.name + ' makes a noise.');
+  }
+}
+
+class Dog extends Animal {
```
Comment (houndci-staging) [inline app/assets/javascripts/es6.js:11]: "Dog" is defined but never used

File: app/assets/javascripts/es6.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
+class Animal {
+  constructor(name) {
+    this.name = name;
+  }
+
+  speak() {
+    console.log(this.name + ' makes a noise.');
```
Comment (houndci-staging) [inline app/assets/javascripts/es6.js:7]: Expected indentation of 6 space characters but found 4.<br>Unexpected console statement.<br>Strings must use doublequote.

File: app/assets/javascripts/es6.js [hunk 7]
```diff
@@ -0,0 +1,23 @@
+class Animal {
+  constructor(name) {
+    this.name = name;
```
Comment (houndci-staging) [inline app/assets/javascripts/es6.js:3]: Expected indentation of 6 space characters but found 4.

**Comment by teoljungberg** [comment]
This is to test that https://github.com/thoughtbot/hound-eslint/pull/13 works as expected, so we still comment on violations

**Pull request closed without merging**